src/ray/core_worker/core_worker.cc

```diff
@@ -2451,9 +2451,8 @@ Status CoreWorker::CreatePlacementGroup(
            << ". It is probably "
               "because GCS server is dead or there's a high load there.";
     return Status::TimedOut(stream.str());
-  } else {
```

Comment:
clang-tidy has an item for this, which encourages early return to reduce indentation.

Reply:
Reference: https://clang.llvm.org/extra/clang-tidy/checks/readability/else-after-return.html